library/friendica

```diff
@@ -0,0 +1,49 @@
+# This file is generated via https://github.com/friendica/docker/blob/617b12fd7f1845a18230bd9c06ea726c89905b2c/generate-stackbrew-library.sh
+
+Maintainers: Friendica <info@friendi.ca> (@friendica), Philipp Holzer <admin@philipp.info> (@nupplaphil)
+GitRepo: https://github.com/friendica/docker.git
+
+Tags: 2019.06-apache, apache, stable-apache, production-apache, 2019.06, latest, stable, production
+Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
+GitCommit: ae608b6380adb65075843c8294d52af769087ce8
+Directory: 2019.06/apache
+
+Tags: 2019.06-fpm, fpm, stable-fpm, production-fpm
+Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
+GitCommit: ae608b6380adb65075843c8294d52af769087ce8
+Directory: 2019.06/fpm
+
+Tags: 2019.06-fpm-alpine, fpm-alpine, stable-fpm-alpine, production-fpm-alpine
+Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
+GitCommit: ae608b6380adb65075843c8294d52af769087ce8
+Directory: 2019.06/fpm-alpine
+
+Tags: 2019.09-dev-apache, dev-apache, 2019.09-dev, dev
+Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
+GitCommit: ae608b6380adb65075843c8294d52af769087ce8
+Directory: 2019.09-dev/apache
+
+Tags: 2019.09-dev-fpm, dev-fpm
+Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
+GitCommit: ae608b6380adb65075843c8294d52af769087ce8
+Directory: 2019.09-dev/fpm
+
+Tags: 2019.09-dev-fpm-alpine, dev-fpm-alpine
+Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
+GitCommit: ae608b6380adb65075843c8294d52af769087ce8
+Directory: 2019.09-dev/fpm-alpine
+
+Tags: 2019.09-rc-apache, rc-apache, 2019.09-rc, rc
```

Comment:
Likewise, is "2019.09-rc" a specific release? (not seeing anything at https://github.com/friendica/friendica/tags?)

Reply:
It's a development branch where we only merge fixes towards the `2019.09` release that will have a tag.